aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -1113,24 +1113,37 @@ For more information on Blazor startup, see <xref:blazor/fundamentals/startup>.
 
 :::moniker range=">= aspnetcore-6.0"
 
-## Delay the appearance of the reconnection UI
+## Control when the reconnection UI appears
```

Comment:
Can't comment on this section, I have no idea how the reconnection UI works.

Reply:
Oh, ok ... thanks. Let see if Mackinnon can at least help me triage or comment on it.